spec/models/credit_note_spec.rb

```diff
@@ -125,7 +125,7 @@
     end
 
     it 'returns the list of subscription ids' do
-      expect(credit_note.subscription_ids).to eq([subscription_fee.subscription_id, charge_fee.subscription_id])
+      expect(credit_note.subscription_ids).to contain_exactly(subscription_fee.subscription_id, charge_fee.subscription_id)
```

Comment:
Layout/LineLength: Line is too long. [123/120] (https://rubystyle.guide#max-line-length)